src/views/studio/MyVideosView/EmptyVideosView.tsx

```diff
@@ -87,38 +92,39 @@ const MessageView = styled.div`
 `
 
 // for tabs
-export const EmptyVideos = () => {
+export const EmptyVideos: React.FC<{ text: string }> = ({ text }) => {
   return (
     <Container>
-      <WellIllustration />
+      <TheaterMaskIllustration />
       <Message>
-        <Text variant="h5">No videos found...</Text>
+        <Text secondary variant="body2">
+          {text}
+        </Text>
+        <Button icon={<SvgGlyphAddVideo />} to={absoluteRoutes.studio.editVideo()} size="large">
+          Upload video
+        </Button>
       </Message>
-      <Button icon={<SvgGlyphAddVideo />} to={absoluteRoutes.studio.editVideo()} size="large">
-        Upload video
-      </Button>
     </Container>
   )
 }
 
 const Container = styled.div`
-  margin: ${sizes(20)} auto 0;
   display: grid;
   place-items: center;
   > svg {
-    max-width: 650px;
+    max-width: 100%;
+  }
+  ${media.small} {
+    margin: ${sizes(20)} auto 0;
   }
 `
 
 const Message = styled.div`
-  display: flex;
-  flex-direction: column;
+  position: relative;
+  top: -256px;
+  display: grid;
+  gap: ${sizes(4)};
```

Comment:
```suggestion
  gap: ${sizes(4)};
  justify-items: center;
```
Maybe it's just a matter of preference, but I think the button should have always the same width

Reply:
This line should fix what @kdembler brought here https://github.com/Joystream/atlas/pull/598#discussion_r633392594